benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/custom_metric_aggregator.py

```diff
@@ -60,4 +66,17 @@ def json_dump_report(self):
             "average-time-to-first-token": sum(self.time_to_first_token_list)/max(len(self.time_to_first_token_list),1)
         }
         return json.dumps(stats)
+    
+    def dump_to_csv(self):
+        fields = ['success', 'total_request_time', 'time_to_first_token', 'input_tokens', 'output_tokens']
+        now = datetime.now()
+        bucket_name = "jetstream-benchmark-output-bucket-01"
```

Comment:
Can we use the bucket that the user already specified in the TF config here and just store it with a different file name?

Reply:
Agh, missed that, apologies. Done.

Reply:
I still see hardcoded bucket, double checking commit was added in :)

Reply:
Oh I see below - we should delete this line I think

Reply:
Agreed, done.

Reply:
ty!